bin/migrate-create

```diff
@@ -17,10 +18,21 @@ program
   .option('-t, --template-file [filePath]', 'Set path to template file to use for new migrations', path.join(__dirname, '..', 'lib', 'template.js'))
   .option('-e, --extention [extention]', 'Use the given extention to create the file', '.js')
   .option('-g, --generator <name>', 'A template generator function', path.join(__dirname, '..', 'lib', 'template-generator'))
+  .option('--env <name>', 'Use dotenv to load an environment file')
```

Comment:
`[name]`?

maybe specify the default (`'.env'`) here instead of further down as well?

Reply:
As for me, I think it would be nice to load default `.env` file from project root, even without that option, but keep `--env <path>` to specify custom path if needed.

Reply:
Yeah, I agree with that behaviour 👍 that was how I originally imagined as well, as well how db-migrate does it

Reply:
Thats actually a mistake.  I meant to make a helper for this so that it wasn't a copy paste job.  See the other files where it is `[name]`.  Thanks for catching it.